cmd/sonobuoy/app/delete.go

```diff
@@ -52,35 +53,59 @@ func NewCmdDelete() *cobra.Command {
 }
 
 func deleteSonobuoyRun(cmd *cobra.Command, args []string) {
+	s := spinner.New(spinner.CharSets[0], 100*time.Millisecond)
+	s.Color("red")
+
+	if runflags.wait > 0 {
+		s.Start()
+	}
 	cfg, err := deleteFlags.kubeconfig.Get()
 	if err != nil {
+		if deleteFlags.wait > 0 {
+			s.FinalMSG = "Status: Could not get kubernetes configuration!"
+			s.Stop()
+		}
 		errlog.LogError(errors.Wrap(err, "couldn't get kubernetes config"))
 		os.Exit(1)
 	}
 
 	sbc, err := getSonobuoyClient(cfg)
 	if err != nil {
+		if deleteFlags.wait > 0 {
+			s.Suffix = "Status: Could not create Sonobouy Client!"
+			s.Stop()
+		}
 		errlog.LogError(errors.Wrap(err, "could not create sonobuoy client"))
 		os.Exit(1)
 	}
 
 	client, err := sbc.Client()
 	if err != nil {
+		if deleteFlags.wait > 0 {
+			s.Stop()
+		}
 		errlog.LogError(err)
 		os.Exit(1)
 	}
 
 	rbacEnabled, err := deleteFlags.rbacMode.Enabled(client)
 	if err != nil {
+		s.Suffix = "Status: Could not detect RBAC status!"
+		s.Stop()
 		errlog.LogError(errors.Wrap(err, "couldn't detect RBAC status"))
 		os.Exit(1)
 	}
 	deleteopts.EnableRBAC = rbacEnabled
 	deleteopts.Wait = time.Duration(deleteFlags.wait) * time.Minute
 
 	if err := sbc.Delete(&deleteopts); err != nil {
+		s.Suffix = "Status: Failed to delete sonobuoy resources!"
+		s.Stop()
 		errlog.LogError(errors.Wrap(err, "failed to delete sonobuoy resources"))
 		os.Exit(1)
 	}
-
-}
+	if deleteFlags.wait > 0 {
+		s.FinalMSG = "Status: Success!"
```

Comment:
Again, I'd remove this. This one is especially misleading since users will think this means that their tests were successful when this just means that the run completed.